gta-gallery/src/layout/index.jsx

```diff
@@ -92,6 +68,42 @@ const Layout = () => {
     setVersions(versionsArray);
   };
 
+  const getBase64 = (img, callback) => {
+    const reader = new FileReader();
+    reader.addEventListener('load', () => callback(reader.result));
+    reader.readAsDataURL(img);
+  };
+
+  const handleChange = (info) => {
+    const file = info.file;
+    if (file.status === 'uploading') {
+      return setImageValue({ loading: true });
+    }
+    if (file.status === 'done') {
+      getBase64(file.originFileObj, (imageUrl) =>
+        setImageValue({
+          imageUrl,
+          loading: false,
+        })
+      );
+    }
+  };
+
+  const beforeUpload = (file) => {
```

Comment:
This method we have in 2 places. Please move to other file and use the same method all places